Atomic/scan.py

```diff
@@ -55,19 +55,36 @@ def get_scan_info(scanner, scan_type):
             raise ValueError(yaml_error)
 
         self.results_dir = os.path.join(self.results, self.args.scanner, self.cur_time)
-        scan_list = self._get_scan_list()
-        for i in scan_list:
-            self.scan_content[i['Id']] = i.get('input')
-
-        # mount all the rootfs
-        self._mount_scan_rootfs(scan_list)
 
+        # Create the input directory
+        if not os.path.exists(self.chroot_dir):
+            os.makedirs(self.chroot_dir)
         if self.debug:
-            util.write_out("Creating the output dir at {}".format(self.results_dir))
+            util.write_out("Created {}".format(self.chroot_dir))
+
+        if self.args.rootfs is None:
+            scan_list = self._get_scan_list()
```

Comment:
Nope, if unpopulated, it is None.

[bbaude@localhost atomic (atomic_scan_chroot)]$ sudo ./atomic scan foobar
Namespace(_class=<class 'Atomic.scan.Scan'>, all=False, containers=False, debug=False, func='scan', images=False, list=False, rootfs=None, scan_targets=['foobar'], scan_type=None, scanner='openscap', verbose=False)